api/types/license.go

```diff
@@ -86,6 +86,16 @@ type License interface {
 	// SetSupportsResourceAccessRequests sets resource access requests support flag
 	SetSupportsResourceAccessRequests(Bool)
 
+	// GetMonthlyActiveUsers gets the max number of monthly active users
+	GetMonthlyActiveUsers() uint
+	// SetMonthlyActiveUsers sets the max number of monthly active users
+	SetMonthlyActiveUsers(uint)
+
+	// GetIsTrial returns the trial flag
+	GetIsTrial() Bool
+	// SetIsTrial sets the trial flag
+	SetIsTrial(Bool)
```

Comment:
For MAU, since we are only supporting Cloud, I don't think we need to expose MAU information to Teleport (at least not yet).

For `IsTrial`, since we are going to support Cloud and on-prem, we want to expose the license expiration date. For example, an on-prem customer might have a 1 year license (non-trial) that has expired, and we need to display a warning.

Please sync with @pierrebeaucamp and @jimbishopp about this next week.